contents/blog/send-love-to-open-source.md

```diff
@@ -0,0 +1,55 @@
+---
+date: 2022-02-08
+title: Send love to open-source projects on Valentine's Day
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/send-love-to-open-source.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Open source"]
+---
+
+Valentine's Day is coming up and while PostHoggers around the world will be using the occasion to give thanks to their loved ones, we'd also like to share our love for the open-source community that helps power PostHog. Here's how:
```

Comment:
Valentine's Day is coming up we'd like to share our love for the open-source community that helps power PostHog! Here's how: